packages/cli/lib/DevServerManager.js

```diff
@@ -86,6 +93,7 @@ class DevServerManager {
             debug: this.debug,
             httpClient,
             projectConfig,
+            requestPorts,
```

Comment:
Not a blocker - I wonder if this is something that we should be passing down to the servers or if they should just import it directly from local-dev-lib. They will likely already have local-dev-lib as a dep anyways.

Reply:
Yeah I thought about this a bit. I ended up going with this approach because the CLI is responsible for starting the port manager server. So when the server gets it from the DevServerManager, it comes with a guarantee that the port manager is actually running. In practice though I don't think it would really make a difference, so not opposed to changing this.